exchange/events_test.go

```diff
@@ -0,0 +1,120 @@
+package exchange
+
+import (
+	"testing"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/stretchr/testify/assert"
+)
+
```

Comment:
You've put together some nice integration tests to obtain coverage of `modifyBidsForEvents` and helper methods `isModifyingVASTXMLAllowed` and `modifyBidVAST`. Thoughts on whether we should add a unit test for `modifyBidsForEvents`? I imagine you figured it would be a pain to test via UT because of the complexity around creating the seat bid function inputs.

Reply:
I've left it out because its behavior and parameters changed with the spec changes and I started leaning on the exchangeJSON tests to validate them instead.
Since those cover the code anyway, I feel the specialized UT would add less value now.